client/src/components/categorical/category/index.tsx

```diff
@@ -633,6 +641,7 @@ const CategoryValueList = React.memo(
     colorData,
     // @ts-expect-error ts-migrate(2339) FIXME: Property 'colorTable' does not exist on type '{ ch... Remove this comment to see the full error message
     colorTable,
+    colorMode,
```

Comment:
Looks like this is leading to a linting error: 
```src/components/categorical/category/index.tsx(558,5): error TS2339: Property 'colorMode' does not exist on type '{ children?: ReactNode; }'.
```

Go ahead and copy the above `@ts-expect-error` and we'll lint this together tomorrow